src/server/server_family.cc

```diff
@@ -395,6 +388,15 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
       SetMaxClients(listeners_, res.value());
     return res.has_value();
   });
+  config_registry.RegisterMutable("slowlog_log_slower_than", {});
```

Comment:
similarly you will need to register a callback that updates this parameter for all serverstate objects